spec/membership_comparison_spec.rb

```diff
@@ -16,6 +16,9 @@
   let(:suggestion) { { position: mp, term: term42, party: liberal, district: pontiac } }
   let(:suggestion41) { { position: mp, term: term41, party: liberal, district: pontiac } }
   let(:suggestion_without_party) { { position: mp, term: term42, party: { id: nil }, district: pontiac } }
+  let(:suggestion_speaker) do
```

Comment:
I'm a little worried that this is going to be confusing/misleading to people working with this in the future, or using these tests to understand the code. The speaker of a legislature generally shouldn't be modelled as a subclass of a legislator — it's a distinct position in its own right. Sometimes, however, this _is_ currently modelled like that in Wikidata, so using it as the test case here makes it seem as if we're explicitly testing something about how we handle a member who is also the speaker, which I don't think is what you're doing here. I think it would be much better to make test case explicitly reflect the scenario we're actually working with — i.e. term-specific legislative items.

Reply:
I have updated the items we're using the spec setup in commit 89724b2. Do you think this is enough for now?

Reply:
I guess this depends on how realistic we want to make the tests, or how much it matters that future readers can make sense of them. I'm finding a lot of them quite confusing now, as they explicitly claim they're relating to a member of the Lok Sabha, but then have seats and parties etc that are Canadian. I think I'd have been more minded to either fake a new position that was "member of the 42nd Canadian Parliament", or the like if we wanted to keep going with the current test data (I don't think the tests actually require these to be real Wikidata items, do they?), or else ignore all the Canadian ones entirely, and have a new setup entirely for the India case.

It's largely your call (or the reviewer's!) as to how much that matters, but my general preference is to make test cases as clear and understandable as possible, as these are, in many ways, the de facto product specifications.